src/shared/lib/featureFlags/FeatureFlagsService.ts

```diff
@@ -61,6 +61,18 @@ export class FeatureFlagsService implements IFeatureFlagsService {
   }
 
   evaluateFlag(flag: string): boolean {
+    // Dev environment overrides
+    if (__DEV__) {
+      const devOverrides: Record<string, boolean> = {
+        'enable-better-drawing-image-sizing': true,
+        // Add other feature flags here as needed for dev environment
+      };
+
+      if (flag in devOverrides) {
+        return devOverrides[flag];
+      }
```

Comment:
Instead of doing this approach, can we do something similar to the other repos in which we have default values for each of the flags? This way we don't rely on manual overrides for each env, since this would cause inconsistencies in envs such as UAT, for example.

Take [admin repo's implementation](https://github.com/ChildMindInstitute/mindlogger-admin/blob/develop/src/shared/hooks/useFeatureFlags.const.ts) as example.